### PR TITLE
User Group - Adding Gold Coast page

### LINKED ISSUE #2074: ✨ UG - Add Gold Coast page
**From**: @camillars 
**Sent**: Wednesday, January 17, 2024 7:30 AM
**To**: @jeoffreyfischer 
**Cc**: @PennyWalker @amankumarrr 
**Subject**: Gold Coast UG page missing from ssw.com.au

**Is your feature request related to a problem? Please describe.**
We are missing a page on the website for the Gold Coast UG [https://www.meetup.com/gold-coast-azure-net-user-group/events/298451887/](https://www.meetup.com/gold-coast-azure-net-user-group/events/298451887/)

**Describe the solution you'd like**
On the website, add a new page for Gold Coast and add it to the MegaMenu.

**Tasks**

- [x] Create the Gold Coast UG page.
- [x] After the page is created, add it to the MegaMenu.
- [x] Get it checked by another Dev.

**Acceptance criteria**

- [x] Reply 'Done' to the email and also @mention Camilla in the PBI with 'Done'

### PR DESCRIPTION
**Pull request by amankumarrr**
Fixed: #2074
Affected route: `/netug/gold-coast`

- Fixed the vertical alignment for GitHub Discussion button 
image

- Fixed hydration issue on the page 
- Fixed the alignment of the Tip field for tina  
image

**Figure: Tip was not wrapping onto the next row**

- Hid Twitter widget after having discussion with @wicksipedia because it was not showing any timeline and it's a know issue from Twitter. 
Twitter-Not-Showing

**Figure: Twitter was not showing any tweets**

### CODE REVIEW

**Comment by camillars** [comment]
@amankumarrr is the SEO for this page consistent with the others?

cheers.

**Comment by amankumarrr** [comment]
> @amankumarrr is the SEO for this page consistent with the others?
> 
> cheers.

@camillars, Yes, it is consistent as per Brisbane's User Group page. Though the title is too long, can you suggest a better title for this page. 

image

**Figure: Showing breadcrumb on Gold Cold page**

**Comment by jeoffreyfischer** [comment]
Great work! Just a few comments: 

1. Can you change the breadcrumb
       From 
       "SSW - Gold Coast Full Stack User Group | .NET development, Azure, Power Platform, general DevOps and ALM 
       strategies, Angular, React, Blazor, and MAUI"
       To 
       "Gold Coast's .NET User Group"

2. Can you add the link to this page in the /netug page?
3. Can you add a link to this page in the MegaMenu too (as per issue #2074)

**Comment by wicksipedia** [comment]
Small issues I've noticed:

CleanShot 2024-01-29 at 12 00 39

1. Change from:
   
   Gold-coast Time
   
   To:
   
   Gold Coast time
   
CleanShot 2024-01-29 at 12 04 07

2. Community - tweets section is blank (blokcked by X?)
3. Community - facebook feed constantly spins
4. Community - content overlaps on smaller screens (should be more responsive)

**Comment by amankumarrr** [comment]
> Small issues I've noticed:
> 
> CleanShot 2024-01-29 at 12 00 39
> 
> 1. Change from:
>    Gold-coast Time
>    To:
>    Gold Coast time
> 
> CleanShot 2024-01-29 at 12 04 07
> 
> 2. Community - tweets section is blank (blokcked by X?)
> 3. Community - facebook feed constantly spins
> 4. Community - content overlaps on smaller screens (should be more responsive)

I'm unable to replicate the issue where the Facebook widget continuously spins when the `ublock` ad blocker is turned on.

File: components/usergroup/sections/community.tsx [hunk 1]
```diff
@@ -28,9 +28,9 @@ export const CommunitySection = (props: CommunitySectionProps) => {
           >
             {props.heading || "Community"}
           </h2>
-          <div className="w-full grid-cols-3 gap-6 md:grid">
+          <div className="flex w-full flex-col items-center justify-center gap-x-6 md:flex-row">
             <div
```
Comment (wicksipedia) [inline components/usergroup/sections/community.tsx:33]: i'd comment out the div with a comment as to why instead of hiding it with css

**Review by wicksipedia — APPROVED** [review]
apart from the above comment, lgtm

**Pull request merged**